recipes/homog2d/all/conandata.yml

```diff
@@ -0,0 +1,13 @@
+sources:
+  "2.9":
+    url: "https://github.com/skramm/homog2d/archive/refs/tags/v2.9.tar.gz"
+    sha256: "7784237154fd0c1beea5b1ed4349e910ce86ad463d21f4f21019f553834c1dad"
+
+patches:
+  "2.9":
+    - patch_file: "patches/0001-include-array.patch"
```

Comment:
This project is super active, you should consider opening a PR to fix these issues and ask for a new release ❤️

Reply:
@prince-chrismc 
I agree with you.